newrelic/structures_newrelic_synthetics_simple_browser_monitor.go

```diff
@@ -164,6 +165,9 @@ func buildSyntheticsSimpleBrowserMonitorRuntimeAndDeviceEmulationUpdateStruct(d
 		if runtimeTypeVersionOk {
 			simpleBrowserMonitorUpdateInput.Runtime.RuntimeTypeVersion = synthetics.SemVer(runtimeTypeVersion.(string))
 		}
+	} else {
+		simpleBrowserMonitorUpdateInput.Runtime.RuntimeType = ""
+		simpleBrowserMonitorUpdateInput.Runtime.RuntimeTypeVersion = ""
```

Comment:
@pranav-new-relic DO NOT add this change to the update function for Simple Browser Monitors. Why?

- Technically, a simple browser monitor cannot exist in the legacy runtime. Today, if such a monitor is created with legacy runtime attributes too from Terraform, a drift is shown, indicating runtime values of the new runtime.
- Because of this behaviour, in the update function, if we provision downgrading from the new runtime to the legacy runtime, the update request accepts runtime values as empty strings and sets them in the UI as well, against the "Legacy Runtime" checkbox, with the `terraform apply` for update. However, because of the point stated above, if the customer runs a `terraform plan` on their configuration after this, they would still see that a drift is shown showing new runtime values; and upon trying to fix the drift and running a `terraform apply`, they would not be able to, as there would be no changes matched against the API :) however, the UI would continue to display the mode as "Legacy Runtime" though the monitor exists in the new runtime.
- To overcome this limitation, it would be ideal to leave this part of the code unchanged (i.e. revert these changes). This would ensure an empty runtime is not populated in the update request when the runtime attributes in the configuration of the customer are empty (or do not exist), thereby keeping the mode of the monitor "New Runtime" in the UI. This would still not fix the problem with the drift, as a drift would be shown asking the customer to move to the values of the new runtime as fetched by the read query, but this would atleast not lead to a false assumption by the customer that the monitor is actually going into the legacy runtime.